source/Engine.cpp

```diff
@@ -2738,3 +2738,16 @@ void Engine::EmplaceStatusOverlay(const shared_ptr<Ship> &it, Preferences::Overl
 	statuses.emplace_back(it->Position() - center, it->Shields(), it->Hull(),
 		min(it->Hull(), it->DisabledHull()), max(20., width * .5), type, alpha);
 }
+
+
+
+void Engine::HandleEvents()
+{
+	for(const auto &event : events)
+		if(event.Type() ^ ShipEvent::DISABLE && event.Target()->CanBeCarried())
```

Comment:
I believe we want a bitwise and here, not a xor.
```suggestion
		if(event.Type() & ShipEvent::DISABLE && event.Target()->CanBeCarried())
```

Reply:
Okay I'm genuinely getting worried about how much I'm getting wrong here. I should not be coding with a fever.